### ISSUE TITLE
electrumx_rpc peers "Host" column truncated

### ISSUE THREAD

**Issue by shsmith** [kyuupichan/electrumx#748]
I have been filtering the output of 'electrumx_rpc peers' through a script to check blacklist status. This is not working for some malicious peers because the "Host" column in the rpc output is being truncated. Could we get that column widened or at least not truncated at the limit?

https://github.com/spesmilo/electrum/issues/5091#issuecomment-464366843

**Comment by shsmith** [comment]
```
diff --git a/electrumx/lib/text.py b/electrumx/lib/text.py
index 8546eb9..ca811ca 100644
--- a/electrumx/lib/text.py
+++ b/electrumx/lib/text.py
@@ -67,7 +67,7 @@ def peers_lines(data):
         features = item['features']
         hostname = item['host']
         host = features['hosts'][hostname]
-        yield fmt.format(hostname[:30],
+        yield fmt.format(hostname,
                          item['status'],
                          host.get('tcp_port') or '',
                          host.get('ssl_port') or '',

```

**Comment by kyuupichan** [comment]
This will lead to columns not being vertical, no?

**Comment by shsmith** [comment]
Yes, but we already have problems like that with the protocol version field. The output of this RPC is hard to parse because of optional columns, lack of delimiters and selective truncation. It would be nice if there was an option to get all RPC output in json format, or at least get delimiters between fields.

Now that https://github.com/kyuupichan/electrumx/pull/753 is merged, I don't think we need this fixed.
